src/app/static/src/tests/mocks.ts

```diff
@@ -0,0 +1,26 @@
+import {RunReportMetadata, RunWorkflowMetadata} from "../js/utils/types";
+
+export const mockRunReportMetadataResponse = (props: Partial<RunReportMetadata> = {}): RunReportMetadata => {
+    return {
```

Comment:
I think `mockRunReportMetadata` (since the return type is just `RunReportMetadata`